core-java-modules/core-java-string-algorithms-3/src/test/java/com/baeldung/emailvalidation/EmailValidationUnitTest.java

```diff
@@ -0,0 +1,89 @@
+package com.baeldung.emailvalidation;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class EmailValidationUnitTest {
+
+    @Test
+    public void testUsingEmailValidator() {
+
+        String emailaddress = "username@domain.com";
+
+        boolean result = EmailValidation.usingEmailValidator(emailaddress);
+
+        assertEquals(result, true);
+    }
+
+    @Test
+    public void testUsingSimpleRegEx() {
+
+        String emailaddress = "username@domain.com";
+
+        boolean result = EmailValidation.usingSimpleRegEx(emailaddress);
+
+        assertEquals(result, true);
+    }
+
+    @Test
+    public void testUsingStrictRegEx() {
+
+        String emailaddress = "username@domain.com";
+
+        boolean result = EmailValidation.usingStrictRegEx(emailaddress);

```

Comment:
Likewise, no need for all these blank lines and can't we just make a call to the EmailValidation in the assert method directly to make the code more concise?